src/apps/workspaces/router.py

```diff
@@ -33,3 +39,22 @@
         **DEFAULT_OPENAPI_RESPONSE,
     },
 )(workspace_applets)
+
+# Remove manager access from a specific user
+router.post(
+    "/removeAccess",
+    response_model=None,
+    responses={
+        status.HTTP_200_OK: {"model": None},
+        **DEFAULT_OPENAPI_RESPONSE,
+    },
+)(workspace_remove_manager_access)
+
+applet_router.post(
+    "/removeAccess",
+    status_code=status.HTTP_200_OK,
+    responses={
+        **DEFAULT_OPENAPI_RESPONSE,
+        **AUTHENTICATION_ERROR_RESPONSES,
+    },
+)(applet_remove_respondent_access)
```

Comment:
Why do you import the router into this file?
Actually, it is effortless to have the circular import using this approach. If this happens you should think about how to divide modules or replace some stuff instead of importing the router from another domain into this one :)

Reply:
The requirement, that remove_respondent_access would be inside applet, is something we already talked about. Do you suggest to move api inside applet/api.py, instead of importing router? or any other options?

Reply:
Just think about how you are doing with your imports. You have 2 separate apps. We have different apps to segregate interfaces. Now you are gathering them again in terms of routers.
@Tom1204 your turn. Please, could you check this and tell is it acceptable?